tests/chainer_tests/link_hooks_tests/test_weight_standardization.py

```diff
@@ -0,0 +1,185 @@
+import unittest
+
+import numpy
+import pytest
+
+import chainer
+from chainer.backends import cuda
+from chainer.link_hooks.weight_standardization import WeightStandardization
+import chainer.links as L
+from chainer import testing
+from chainer.testing import attr
+
+
+class TestExceptions(unittest.TestCase):
+
+    def setUp(self):
+        self.x = chainer.Variable(numpy.ones((10, 5), dtype=numpy.float32))
+        self.layer = L.Linear(5, 20)
+
+    def test_wrong_weight_name(self):
+        wrong_Weight_name = 'w'
+        hook = WeightStandardization(weight_name=wrong_Weight_name)
+        with pytest.raises(ValueError):
+            self.layer.add_hook(hook)
+
+    def test_raises(self):
+        with pytest.raises(NotImplementedError):
+            with WeightStandardization():
+                self.layer(self.x)
+
+    def test_invalid_shaped_weight(self):
```

Comment:
Does WeightStandardization hook have some limitations of the shape of a weight?

Reply:
There is no limitations. Thanks.